### ISSUE TITLE
New Context with viewport set to None does not behave as in Playwright

### ISSUE THREAD

**Issue by mardukbp** [MarketSquare/robotframework-browser#2054]
**Describe the bug**
Given Chromium is started passing the parameter `--start-maximized`
When calling the keyword `New Context    viewport=${None}`
Then the window is not maximized.

The corresponding code in Node.JS works as expected:

```js
const playwright = require('playwright');

(async () => {
  const browser = await playwright.chromium.launch({args: ["--start-maximized"], headless: false});
  const context = await browser.newContext({viewport:null});
  const page = await context.newPage();
  await page.goto('https://browser.fi/');
  await browser.close();
})();
```

**To Reproduce**

Execute the following Robot Framework script:

```robot
*** Settings ***
Library    Browser

*** Test Cases ***
Open Chromium Maximized
    New Browser    chromium    headless=false    args=["--start-maximized"]
    New Context    viewport=${None}
    New Page       https://browser.fi
```

**Expected behavior**
Using Browser Library the window should be maximized, just like in Playwright.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chromium
 - Version: 102.0.5005.40 

**Additional context**

**Comment by aaltat** [comment]
@allcontributors please add @mardukbp for bugs

**Comment by aliasgerkw** [comment]
@mardukbp @aaltat Any ETA on this?
Facing same issue while passing --start-maximized as args but browser is not maximizing on launch.

**Keyword**
```
start the browser
[Arguments]         ${URL}=${BU_URL}        ${LOGIN_CLIENT_ID}=${CLIENT_ID}
    New Browser    browser=${BROWSER}
    ...       headless=${HEADLESS_MODE}
    ...       args=["--start-maximized"]
    New Context
    New Page        ${URL}/login?clientId=${LOGIN_CLIENT_ID}
```
    
 **Logs**
<img width="1328" alt="Screenshot 2022-06-28 at 3 04 54 PM" src="https://user-images.githubusercontent.com/51911147/176146274-095dcd75-ee74-429a-bc8b-aec67996af11.png">

**Comment by aaltat** [comment]
This project, as many other open source projects, is run by people donating their free time without compensation. Therefore this project does not provide estimates when something will be done. If you have an urgent need, usually the fastest to way to get it fixed, is to provide a PR to deal the problem.

**Comment by aliasgerkw** [comment]
Sure @aaltat, intention was to check if someone looked into it. Will try to solve it.

**Comment by Snooz82** [comment]
This is not trivial and i would really like to discuss the solution first, before anyone starts working on that.

Atm we do not send options to pw if they are set to ${NONE}, which means Option==${None} is not possible right now.

**Comment by Snooz82** [comment]
Will hopefully be fixed by #2308

**Comment by aliasgerkw** [comment]
Thanks @Snooz82
Do we need to pass `args=["--start-maximized"]` in New Browser as argument once this is available? or any workaround?

**Comment by Snooz82** [comment]
@aliasgerkw 
As far as i know, yes. you have to set that chrome argument.
I can not test it, because on mac fullscreen does not work.
But @mardukbp tested it on windows.

**Comment by Snooz82** [comment]
Fixed by #2308
